.github/workflows/ci.yaml

```diff
@@ -121,3 +121,17 @@ jobs:
           args: release --rm-dist
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      - name: Build Python package
+        run: |
+          # clean package built for go client
+          rm -rf python/dist
+          # set version
+          sed -i -E "s/0.0.1/${GITHUB_REF_NAME:1}/" setup.py
```

Comment:
What does the `:1` expansion do here?

Reply:
Removes first character. The tag is `v1.2.3`.